ci/analyze.sh

```diff
@@ -30,7 +30,13 @@ function follow_links() (
 SCRIPT_DIR=$(follow_links "$(dirname -- "${BASH_SOURCE[0]}")")
 SRC_DIR="$(cd "$SCRIPT_DIR/../.."; pwd -P)"
 FLUTTER_DIR="$SRC_DIR/flutter"
+
```

Comment:
Given we don't follow this elsewhere, leave a comment that this is to use this from XYZ

Reply:
Done.